<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>190</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Node</string>
  </property>
  <widget class="QLineEdit" name="x_coordinate">
   <property name="geometry">
    <rect>
     <x>68</x>
     <y>40</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="y_coordinate">
   <property name="geometry">
    <rect>
     <x>68</x>
     <y>68</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="z_coordinate">
   <property name="geometry">
    <rect>
     <x>68</x>
     <y>96</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="ID_Input">
   <property name="geometry">
    <rect>
     <x>68</x>
     <y>12</y>
     <width>113</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>40</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>12</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Node ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>68</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>y:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>12</x>
     <y>96</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>z:</string>
   </property>
  </widget>
  <widget class="QToolButton" name="AcceptButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>122</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>ID_Input</tabstop>
  <tabstop>x_coordinate</tabstop>
  <tabstop>y_coordinate</tabstop>
  <tabstop>z_coordinate</tabstop>
  <tabstop>AcceptButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
